_reports/akshatkarani.md

```diff
@@ -0,0 +1,149 @@
+---
+categories:
+  - GSoC
+  - 2019
+  - StatusReport
+date: 2019-08-25
+student: Akshat Karani
+organisation: coala
+organisation_link: https://coala.io
+project: Next Generation Action System
+project_link: https://summerofcode.withgoogle.com/projects/#5450946933424128
+tarball: https://github.com/akshatkarani/GSoC/raw/master/akshatkarani.tar.gz
+sha256sum: 5460a8042082690c2353f487ff9a6fd0a2d4ade1597baaf7e2b74b1633dcd5f6
+mentors: >
+  [Abhinav Kaushlya](https://github.com/abhishalya), [Vamshi Krishna](https://github.com/Vamshi99) & [Kriti Rohilla](https://github.com/kriti21)
```

Comment:
This line is too long, better to split this.

Reply:
The list of mentors for this project was modified.  Please remove the extra mentor